VMBackup/main/fsfreezer.py

```diff
@@ -248,6 +238,18 @@ def freeze_safe(self,timeout):
             self.logger.log(error_msg, True, 'Error')
         return freeze_result,timedout
 
+    def releaseFileLock(self):
+        if (self.isAquireLockSucceeded == True):
+            try:
+                fcntl.lockf(self.safeFreezelockFile, fcntl.LOCK_UN)
+                self.safeFreezelockFile.close()
+            except Exception as e:
+                self.logger.log("Failed to unlock:  "+ str(e),True)
```

Comment:
use this format for stack trace...you can refer other code locations

str(e), traceback.format_exc()

Reply:
updated the pr